.github/workflows/prod-deploy.yml

```
@@ -0,0 +1,33 @@
+    push:
+        branches: [main]
+
+jobs: 
+    deploy:
+        runs-on: ubuntu-latest
+        environment: Production
+
+        steps:
+        - name: Checkout repo
+          uses: actions/checkout@v4
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)